pkg/runner/runner_test.go

```diff
@@ -115,16 +98,20 @@ func TestRunEvent(t *testing.T) {
 		{"testdata", "matrix-include-exclude", "push", "", platforms, "linux/arm64"},
 		{"testdata", "commands", "push", "", platforms, "linux/arm64"},
 		{"testdata", "workdir", "push", "", platforms, "linux/arm64"},
-		// {"testdata", "issue-228", "push", "", platforms, "linux/arm64"}, // TODO [igni]: Remove this once everything passes
 		{"testdata", "defaults-run", "push", "", platforms, "linux/arm64"},
 		{"testdata", "uses-composite", "push", "", platforms, "linux/arm64"},
+		// {"testdata", "powershell", "push", "", platforms}, // Powershell is not available on default act test runner (yet) but preserving here for posterity
+		// {"testdata", "issue-228", "push", "", platforms, "linux/arm64"}, // TODO [igni]: Remove this once everything passes
+
 	}
 	log.SetLevel(log.DebugLevel)
 
 	ctx := context.Background()
+	secretspath, _ := filepath.Abs("../../.secrets")
+	secrets, _ := godotenv.Read(secretspath)
 
 	for _, table := range tables {
-		runTestJobFile(ctx, t, table)
+		runTestJobFile(ctx, t, table, secrets)
```

Comment:
Currently in order to run tests locally, some require checkout and checkout requires GITHUB_TOKEN there's no way to supply the GITHUB_TOKEN currently, hence why I added this. Those tests will always fail locally unless they are run in Github Actions

Reply:
Weird because they always worked locally

Reply:
I always got a "token parameter is required but not supplied". Where was your GITHUB_TOKEN coming from for testing?

Reply:
To be frank, I've never specified any :)

Reply:
@catthehacker marking this as resolved as I can't reproduce, on my machine it always fails with token not found for Github checkout v2 and the source code for checkout seems this is the correct behavior, it will fail if a github token is not specified. I know you have a code path in there that will skip checkout for certain scenarios, especially around binding, maybe the move to test using volumes exposed a bug?